mw-webapp/src/logic/wayPage/reportsTable/DayReportsTable.tsx

```diff
@@ -33,6 +34,11 @@ export const DayReportsTable = (props: DayReportsTableProps) => {
   const way = props.way;
   const {user} = useGlobalContext();
   const isOwner = user?.uuid === way.owner.uuid;
+  const emptyWay = dayReports.length === 0;
+  const shortISODateToday = DateUtils.getShortISODateValue(new Date());
+  const shortISODate = !emptyWay && DateUtils.getShortISODateValue(dayReports[0].date);
```

Comment:
If you already have shortISODateToday, than current variable is tomorrow?

Reply:
shortISODateToday is today date for check, shortISODate - last date from table